tests/functional_tests/models/mixtral/test_mixtral_conversion.py

```
@@ -0,0 +1,269 @@
+            "--hf-model-id",
+            mixtral_toy_model_path,
+            "--output-dir",
+            str(test_output_dir),
+            "--tp",
+            str(tp),
+            "--pp",
+            str(pp),
+        ]
+
+        try:
+            result = subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)